docs/build/examples/sample-architectures.md

```diff
@@ -8,7 +8,7 @@ description: Learn how you might build an app on top of Filecoin.
 No matter what you wish to build on Filecoin, you'll generally need a few basic elements:
 
 - **A node.** Each Filecoin-based service or application will need to use at least one Filecoin node that maintains consensus. All interactions with the network must flow through an up-to-date node: sending and receiving market deals, sending and receiving data, and more. You can host this yourself, or choose a [hosted node](../start-building/interacting-with-the-network.md) option.
-- **An API.** Depending on language, and your preference for more granular or high-level commands, you can either directly use the node's go RPC API, or use one of the available [JS API client libraries](../developer-tools/wallets-signing-tools-api-clients.md#api-clients).
+- **An API.** Depending on language, and your preference for more granular or high-level commands, you can either directly use the node's go RPC API, or use one of the available [JS API client libraries](../tools/).
```

Comment:
```suggestion
- **An API.** Depending on language, and your preference for more granular or high-level commands, you can either directly use the node's go RPC API, or use one of the available [JS API client libraries](../tools/).
```

@cwaring what do we do in situations like this, where the landing page for `/build/tools` is `README.md`. I imagine having this link as `../tools/README.md` would cause 404s to pop up on Netlify's end.